frontend_scv/lib/widgets/contact/ContactList_item.dart

```diff
@@ -0,0 +1,35 @@
+//This is the UI counterpart of ContactList from models.
+
+import 'package:flutter/material.dart';
+import 'package:unison/effects/glow_on_hover.dart';
+import 'package:unison/models/contactList.dart';
+import 'package:unison/screens/view_contact_screen.dart';
+
+class ContactListItem extends StatelessWidget {
+  final ContactList list;
+  ContactListItem(this.list);
+
+  @override
+  Widget build(BuildContext context) {
+    Widget contactListItem = GestureDetector(
+      onTap: () {
+        print('A tap');
```

Comment:
???

Reply:
An unnecessary print, should be removed in the future.